extkingpin/pathorcontent.go

```diff
@@ -109,6 +111,13 @@ func WithEnvSubstitution() Option {
 	}
 }
 
+// WithHidden allows you to override default hidden option and keep the flags hiddem.
```

Comment:
nit:
```suggestion
// WithHidden allows you to override the default option and keeps the flag hidden.
```

Reply:
Done in https://github.com/efficientgo/tools/pull/14/commits/6ce36fe0ac57396dc927d017605506d4cbffec4e